src/geometry/manifold/cont_mdiff.lean

```diff
@@ -925,6 +925,13 @@ begin
   { simp only [written_in_ext_chart_at, (∘), mem_ext_chart_source, e.left_inv, e'.left_inv] }
 end
 
+/-- See note [comp_of_eq lemmas] -/
+lemma cont_mdiff_within_at.comp_of_eq {t : set M'} {g : M' → M''} {x : M} {y : M'}
```

Comment:
Are we missing `continuous_within_at.comp_of_eq`?

Reply:
There are many of these versions missing. I don't want to add all the versions to mathlib3, since that will just cause more forward-porting obligations. I do want to go through mathlib4 and add them systematically.